packages/zowe-explorer/package.json

```diff
@@ -1767,9 +1767,8 @@
     "compile-web": "webpack --target webworker --entry ./src/web/extension.ts --output-path ./out/src/web",
     "markdown": "markdownlint CHANGELOG.md README.md",
     "lint:pretty": "prettier --check .",
-    "lint": "yarn lint:src && yarn lint:tests && yarn lint:pretty && yarn markdown",
-    "lint:src": "tslint --format stylish --exclude [\"**/__tests__/**/*.ts\",\"node_modules/**/*\"] \"src/**/*.ts\"",
-    "lint:tests": "tslint --format stylish \"**/__tests__/**/*.ts\" --exclude \"node_modules/**/*\" && eslint \"__tests__/**\"",
+    "lint": "yarn lint:src && yarn lint:pretty && yarn markdown",
+    "lint:src": "eslint --format stylish src/**/*.ts",
```

Comment:
Should we keep the separate "lint:tests" script in case anyone was using it?

Reply:
Fixed in new commit 👍